ctapipe_io_magic/__init__.py

```diff
@@ -80,6 +82,75 @@
     DATA_MAGIC_LST_TRIGGER: EventType.SUBARRAY,
 }
 
+class ReportLaserContainer(Container):
+    """ Container for Magic laser parameters """
+    UniqueID = Field(None, 'No.')
+    Bits = Field(None, 'ID')
+    MJD = Field(None, 'Modified Julian Date')
+    BadReport  = Field(bool, 'Bad Report')
+    State = Field(None, 'State')
+    IsOffsetCorrection = Field(bool, 'Is Offset Correction')
+    IsOffsetFitted = Field(bool, 'Is Offset Fitted')
+    IsBGCorrection = Field(bool, 'Is BG Correction')
+    IsT0ShiftFitted = Field(bool, 'Is T0 Shift Fitted')
+    IsUseGDAS = Field(bool, 'Is Use GDAS')
+    IsUpwardMoving = Field(bool, 'Is Upward Moving')
+    OverShoot = Field(nan, 'Over Shoot')
+    UnderShoot = Field(nan, 'Under Shoot')
+    BGSamples = Field(np.float32(np.nan), 'BG Samples')
+    Transmission3km = Field(np.float32(np.nan), 'Transmission at 3 km')
+    Transmission6km = Field(np.float32(np.nan), 'Transmission at 6 km')
+    Transmission9km = Field(np.float32(np.nan), 'Transmission at 9 km')
+    Transmission12km = Field(np.float32(np.nan), 'Transmission at 12 km')
+    Zenith = Field(Angle(np.nan, u.deg), 'Zenith angle', unit=u.deg)
+    Azimuth = Field(Angle(np.nan, u.deg), 'Azimuth angle', unit=u.deg)
+    CloudFWHM = Field(np.float32(np.nan), 'Cloud FWHM')
```

Comment:
CloudHM, HStd, LR, FWHM, Base, Top, TopT, Trans are defined in MReportLaser.h as fixed size tables, but they are not put into a streamer as tables, so I think they are not stored. I checked two random _S_ files and they are all set to 0 or 1, so I think there is no point to read them here

Reply:
I removed the unnecessary parameters.